projects/packages/calypsoify/src/js/mods-gutenberg.js

```diff
@@ -1,45 +1,40 @@
 /* eslint-disable no-var */
-/* global wp, calypsoifyGutenberg */
+/* global calypsoifyGutenberg */
 import '../scss/style-gutenberg.scss';
+import { dispatch, select } from '@wordpress/data';
 
-jQuery( function ( $ ) {
-	// Force fullscreen mode for iframed post editor.
-	if (
-		wp &&
-		wp.data &&
-		wp.data.select( 'core/edit-post' ) &&
-		! wp.data.select( 'core/edit-post' ).isFeatureActive( 'fullscreenMode' )
-	) {
-		wp.data.dispatch( 'core/edit-post' ).toggleFeature( 'fullscreenMode' );
-	}
+// Force fullscreen mode for iframed post editor.
+if (
+	select( 'core/edit-post' ) &&
+	! select( 'core/edit-post' ).isFeatureActive( 'fullscreenMode' )
+) {
+	dispatch( 'core/edit-post' ).toggleFeature( 'fullscreenMode' );
+}
 
-	// Force fullscreen mode for iframed site editor.
-	if (
-		wp &&
-		wp.data &&
-		wp.data.select( 'core/edit-site' ) &&
-		! wp.data.select( 'core/edit-site' ).isFeatureActive( 'fullscreenMode' )
-	) {
-		wp.data.dispatch( 'core/edit-site' ).toggleFeature( 'fullscreenMode' );
-	}
+// Force fullscreen mode for iframed site editor.
+if (
+	select( 'core/edit-site' ) &&
+	! select( 'core/edit-site' ).isFeatureActive( 'fullscreenMode' )
+) {
+	dispatch( 'core/edit-site' ).toggleFeature( 'fullscreenMode' );
+}
 
-	var editPostHeaderInception = setInterval( function () {
-		var $closeButton = $( '.edit-post-fullscreen-mode-close__toolbar a' );
-		if ( $closeButton.length < 1 ) {
-			return;
-		}
-		clearInterval( editPostHeaderInception );
+var editPostHeaderInception = setInterval( function () {
+	var closeButton = document.querySelector( '.edit-post-fullscreen-mode-close__toolbar a' );
+	if ( closeButton.length < 1 ) {
```

Comment:
`querySelector()` returns a node or null, not anything with a length property.
```suggestion
	if ( ! closeButton ) {
```

Reply:
Thanks for these! I've not got a good way to test many of these changes just yet so will make the changes but need to follow up later when I can actually test things.